hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/EmbeddableMappingTypeImpl.java

```diff
@@ -182,7 +182,7 @@ private EmbeddableMappingTypeImpl(
 				updatable = componentProperty.isUpdateable();
 			}
 			this.aggregateMapping = SelectableMappingImpl.from(
-					bootDescriptor.getOwner().getTable().getName(),
+					bootDescriptor.getOwner().getTable().getQualifiedTableName().render(),
```

Comment:
```suggestion
					bootDescriptor.getOwner().getTable().getQualifiedName( creationContext.getSqlStringGenerationContext() ),
```
We need this in case default schema or default catalog are configured

Reply:
Changed code